charts/sn-platform/templates/broker/broker-cluster.yaml

```diff
@@ -121,7 +121,9 @@ spec:
           {{ end }}
       {{ end }}
     {{- end }}
+    {{- if or .Values.auth.authentication.enabled (and .Values.broker.offload.s3.enabled .Values.broker.offload.s3.secret) (and .Values.broker.offload.azureblob.enabled .Values.broker.offload.azureblob.secret) }}
     vars:
+    {{- end }}
```

Comment:
can we move the {{- end}} to after all env vars are defined.

Reply:
Fixed, thanks